CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Mapbox Android Telemetry
 
+### v8.1.3
```

Comment:
just double-checking that we shouldn't bump the Core module version as well, right?

Reply:
Correct.